boards/nxp/imx8mp_evk/board.yml

```diff
@@ -1,6 +1,6 @@
 board:
   name: imx8mp_evk
-  full_name: i.MX8MP EVK
```

Comment:
Currently the board name is the same with the information in nxp.com, so I don't think we need to update the name.

Reply:
> Currently the board name is the same with the information in nxp.com, so I don't think we need to update the name.

"-EVK" is written on board labels. Also it used for other NXP board naming.
Actually no matter for me, just to use the common approach.